### PR TITLE
Added seeds spec when rspec is installed

### PR DESCRIPTION
**Pull request by prawl**
## Why?
- It's very common to allow our seeds to run and throw an exception, which shouldn't happen.  Shout out to @TylerRockwell for originally writing this spec.

## What Changed?
- Added seeds spec out of the box when you install rspec to prevent failing seeds going forward.

### CODE REVIEW

File: lib/boxcar/app_builder.rb [hunk 1]
```diff
@@ -30,6 +30,11 @@ def create_secrets_example
 
     def generate_rspec
       generate "rspec:install"
+      install_specs
+    end
+
+    def install_specs
```
Comment (dkniffin) [inline lib/boxcar/app_builder.rb]: 👏 This should work well with your other PR.

**Review by dkniffin — COMMENTED** [review]
The only other thing I'd like to see in this is the same thing we need in the other PR: a test in boxcar that runs the specs for the generated project, and ensures it passes.

File: lib/boxcar/app_builder.rb (hunk 1 above)
Comment (prawl) [reply lib/boxcar/app_builder.rb]: Yup, waiting on that one to merge this one.

**Comment by prawl** [comment]
Thanks @dkniffin

**Pull request merged**